docs/UI/Parsley-Sectioning.md

```diff
@@ -0,0 +1,38 @@
+# Parsley Sectioning
+
+## Overview
+Parsley sectioning is toggleable from the "Log Viewing" tab of the "Parsley Settings" menu and is available to task logs only. Toggling the feature will render your task log into expandable sections split up by function and command.
+
+## Feature details
```

Comment:
nit. Would just recommend double checking if you want these headers in title case or in sentence case since it's slightly inconsistent (e.g. `Section status` vs `Jump to Failing Line`)

Reply:
Good one. I'll stick to sentence case.